.changeset/eleven-ligers-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-tile>`: `<rh-tile-group>` will now apply a CSS grid to child `<rh-tile>` elements.
```

Comment:
```Suggestion
`<rh-tile-group>`:  corrected application of grid layout of slotted elements.
```

Any slotted element in this case will be applied to the grid.  Feel like we should be more general here given that the grid won't just apply to `rh-tile` children, but any element that is slotted.

Reply:
Updated in [0a18877](https://github.com/RedHat-UX/red-hat-design-system/pull/1729/commits/0a188776dfed8626abf08f74a6fa17b86494d441).